### PR TITLE
Feature/client and location

### PR DESCRIPTION
**Pull request by 1serioga**

### CODE REVIEW

File: source/includes/_cb.md [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "start_year": "Please enter starting year",
     "working_hours": "Your must provide Working Hours for at least one day in the week",
     "campaign_status": "Invalid campaign status",
-    "client_id": "Client with  ID 0 not found or doesn't belong to this customer"
+    "location-id": "Location with ID 0 not found or doesn't belong to this customer"
```
Comment (ejeliot) [inline source/includes/_cb.md]: Seems to be inconsistency. Is it locstion-id or location_id? The format should match the API in question.

File: source/includes/_clients.md [hunk 2]
```diff
@@ -118,70 +79,17 @@ Delete an existing client. If there are reports associated with this client then
 
 ### HTTP Request
 
-`POST https://tools.brightlocal.com/seo-tools/api/v2/clients/delete`
+`DELETE https://tools.brightlocal.com/seo-tools/api/v1/clients-and-locations/clients/`
```
Comment (ejeliot) [inline source/includes/_clients.md]: Is trailing / necessary?

File: source/includes/_clients.md [hunk 3]
```diff
@@ -60,45 +41,27 @@ Update an existing client. Only supply values you want to update. The rest will
...
-    }
+    "success": true,
+    "client-id": 1
 }
 ```
 
 ### HTTP Request
 
-`POST https://tools.brightlocal.com/seo-tools/api/v2/clients/update`
+`POST https://tools.brightlocal.com/seo-tools/api/v1/clients-and-locations/clients/`
```
Comment (ejeliot) [inline source/includes/_clients.md]: Is trailing / necessary.

File: source/includes/_locations.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+			"location-id": 1,
+			"location-name":"BrightLocal HQ",
+			"client-id":1,
+			"location-reference":"BL1"
+	   }
+    ]
+}
+```
+
+Search for locaions matching a specified search string. The search uses a number of fields including location name, contact firstname, lastname and email address.
```
Comment (ejeliot) [inline source/includes/_locations.md]: Type. locations.

File: source/includes/_locations.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+### HTTP Request
+
+`GET https://tools.brightlocal.com/seo-tools/api/v1/clients-and-locations/locations`
```
Comment (ejeliot) [inline source/includes/_locations.md:223]: Do endpoints need trailing / or not? Here it's not added, elsewhere it is.

Comment (1serioga) [inline source/includes/_locations.md:223]: No, this endspoint without  trailing /

**Pull request merged**